<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEvent</class>
 <widget class="QDialog" name="AddEvent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add event</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Event</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Event details</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <property name="horizontalSpacing">
           <number>10</number>
          </property>
          <property name="verticalSpacing">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Event starts:</string>
            </property>
            <property name="buddy">
             <cstring>editEventStart</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDateTimeEdit" name="editEventStart">
            <property name="displayFormat">
             <string>yyyy-MM-dd HH:mm</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Event ends:</string>
            </property>
            <property name="buddy">
             <cstring>editEventEnd</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDateTimeEdit" name="editEventEnd">
            <property name="wrapping">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::UpDownArrows</enum>
            </property>
            <property name="displayFormat">
             <string>yyyy-MM-dd HH:mm</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Description:</string>
            </property>
            <property name="buddy">
             <cstring>editDescription</cstring>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QPlainTextEdit" name="editDescription">
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="midLineWidth">
             <number>1</number>
            </property>
            <property name="tabChangesFocus">
             <bool>true</bool>
            </property>
            <property name="lineWrapMode">
             <enum>QPlainTextEdit::NoWrap</enum>
            </property>
            <property name="plainText">
             <string/>
            </property>
            <property name="centerOnScroll">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Location:</string>
            </property>
            <property name="buddy">
             <cstring>editLocatioon</cstring>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QPlainTextEdit" name="editLocatioon">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="tabChangesFocus">
             <bool>true</bool>
            </property>
            <property name="lineWrapMode">
             <enum>QPlainTextEdit::NoWrap</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QCheckBox" name="checkAllDay">
            <property name="text">
             <string>All day event</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QDateEdit" name="editAllDay">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="displayFormat">
             <string>yyyy-MM-dd</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Recurrence</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Recurrence</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <property name="horizontalSpacing">
           <number>10</number>
          </property>
          <property name="verticalSpacing">
           <number>5</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Period:</string>
            </property>
            <property name="buddy">
             <cstring>comboRecurrence</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboRecurrence">
            <item>
             <property name="text">
              <string>no recurrence</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>every day</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>every week</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>every month</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>every year</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Repeat:</string>
            </property>
            <property name="buddy">
             <cstring>radioUntil</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QRadioButton" name="radioUntil">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>until</string>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateEdit" name="editUntil">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="displayFormat">
                 <string>yyyy-MM-dd</string>
                </property>
                <property name="calendarPopup">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QRadioButton" name="radioHitCount">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>hit count</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinHitCount">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QRadioButton" name="radioAlways">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>always</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Reminder</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Reminder</string>
         </property>
         <layout class="QFormLayout" name="formLayout_4">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <property name="horizontalSpacing">
           <number>10</number>
          </property>
          <property name="verticalSpacing">
           <number>5</number>
          </property>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkReminder">
            <property name="text">
             <string>Enable reminder</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Remind time:</string>
            </property>
            <property name="buddy">
             <cstring>comboRemind</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="comboRemind">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::DefaultContextMenu</enum>
            </property>
            <item>
             <property name="text">
              <string>on time</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5 minutes before</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10 minutes before</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>15 minutes before</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>30 minutes before</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>other</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Other:</string>
            </property>
            <property name="buddy">
             <cstring>editRemindOther</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDateTimeEdit" name="editRemindOther">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Remind with:</string>
            </property>
            <property name="buddy">
             <cstring>checkTray</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QCheckBox" name="checkTray">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Tray baloon</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkSound">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Sound</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelError">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editEventStart</tabstop>
  <tabstop>editEventEnd</tabstop>
  <tabstop>checkAllDay</tabstop>
  <tabstop>editAllDay</tabstop>
  <tabstop>editDescription</tabstop>
  <tabstop>editLocatioon</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>comboRecurrence</tabstop>
  <tabstop>radioUntil</tabstop>
  <tabstop>editUntil</tabstop>
  <tabstop>radioHitCount</tabstop>
  <tabstop>spinHitCount</tabstop>
  <tabstop>radioAlways</tabstop>
  <tabstop>checkReminder</tabstop>
  <tabstop>comboRemind</tabstop>
  <tabstop>editRemindOther</tabstop>
  <tabstop>checkTray</tabstop>
  <tabstop>checkSound</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkAllDay</sender>
   <signal>toggled(bool)</signal>
   <receiver>editAllDay</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkAllDay</sender>
   <signal>toggled(bool)</signal>
   <receiver>editEventEnd</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkAllDay</sender>
   <signal>toggled(bool)</signal>
   <receiver>editEventStart</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboRecurrence</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>AddEvent</receiver>
   <slot>on_comboRecurrence_currentIndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkReminder</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboRemind</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkReminder</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkTray</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkReminder</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkSound</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddEvent</receiver>
   <slot>on_buttonBox_accepted()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddEvent</receiver>
   <slot>on_buttonBox_rejected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioUntil</sender>
   <signal>toggled(bool)</signal>
   <receiver>editUntil</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioHitCount</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinHitCount</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_comboRecurrence_currentIndexChanged(int)</slot>
  <slot>on_buttonBox_accepted()</slot>
  <slot>on_buttonBox_rejected()</slot>
 </slots>
</ui>
